packages/ember-views/lib/views/text_field.js

```diff
@@ -54,6 +54,8 @@ export default Component.extend(TextSupport, {
     'width'
   ],
 
+  defaultLayout: null,
```

Comment:
Fwiw @rwjblue and I have discussed ignoring `defaultLayout` when there is no template or layout on a component.